### PR TITLE
Increase code coverage 100%.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/NavBar.php [hunk 1]
```diff
@@ -278,9 +279,6 @@ private function initItemsOptions(): void
         if (isset($this->itemsOptions['class'])) {
             $itemsClass = $this->itemsOptions['class'];
             unset($this->itemsOptions['class']);
-            if (is_array($itemsClass)) {
```
Comment (samdark) [inline src/NavBar.php:281]: Is it not needed?

File: src/Widget.php [hunk 2]
```diff
@@ -85,9 +85,6 @@ protected function addOptions(array $options, string $defaultClass): array
         if (isset($options['class'])) {
             $class = $options['class'];
             unset($options['class']);
-            if (is_array($class)) {
```
Comment (samdark) [inline src/Widget.php:88]: Is it not needed?

**Review by samdark — APPROVED** [review]
Overall looks good except two removals.

File: src/NavBar.php (hunk 1 above)
Comment (terabytesoftw) [reply src/NavBar.php:281]: I think it is not necessary, I have thought and not found a case where the options class is an array.

File: src/Widget.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Widget.php:88]: I think it is not necessary, I have thought and not found a case where the options class is an array.

File: src/NavBar.php (hunk 1 above)
Comment (samdark) [reply src/NavBar.php:281]: Well, I can specify class like `['nav', 'dark']`:

```php
$class = ['nav'];
if ($isDarkMode) {
   $class[] = 'dark';
}
echo Html::blaBla['class' => $class];
```

Comment (terabytesoftw) [reply src/NavBar.php:281]: Done revert and add tests.

File: src/Widget.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Widget.php:88]: Done revert and add tests.

**Review by samdark — APPROVED** [review]

**Pull request merged**